src/main/java/PageObjects/LoginPage.java

```diff
@@ -0,0 +1,34 @@
+package PageObjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class LoginPage {
+    private WebDriver localDriver;
+
+    public LoginPage(WebDriver driver){
+        this.localDriver=driver;
+    }
+
+    String logInLink="//a[text()='Log In']";
+    String userMailID="email";
+    String userPassword="password";
+    String nextButton="//button[@type='submit']";
+
+
+
+    public void clickLoginLink(){
+        localDriver.findElement(By.xpath("//a[text()='Log In']")).click();
```

Comment:
You did not use the loginLink variable. replace hardcoding locator with variable.